app/controllers/api/v1/reset_password_controller.rb

```diff
@@ -13,12 +13,12 @@ class ResetPasswordController < ApiController
 
       def create
         # TODO: Log events.
-        return render_json status: :bad_request unless params[:user]
+        return render_error unless params[:user]
```

Comment:
I think it's better to explicitly write the status code for `render_error` even if it's the default.

Reply:
I'm thinking of dropping the `default` in a later PR and always specifying it